src/nls/fi/strings.js

```diff
@@ -624,7 +624,25 @@ define({
     "CMD_TOGGLE_RECENT_PROJECTS"                : "Viimeisimmät projektit",
 
     // extensions/default/WebPlatformDocs
-    "DOCS_MORE_LINK"                            : "Lue lisää"
+    "DOCS_MORE_LINK"                            : "Lue lisää",
+
+    //extensions/default/CodeFolding
+    "COLLAPSE_ALL"                  : "Pienennä kaikki",
```

Comment:
@abose collapse custom regions closes all custom defined region blocks in the current editor. These are defined using comments of the current mode with the keywords `region & endregion` e.g.,:

    \\region
    var point = {x: 1, y: 2};
    \\endregion

With respect to menus,  as far as I know, brackets does not implement submenus yet. Hence I'm not sure how to go about creating the sublime-like sub-menu for code-folding. So did you mean a top level "code folding" entry (just like view/edit), whose menu items are the folding options?

And as for the strings, I have no more changes to make. Are you happy with the code-folding strings present in the root string? In addition, do let me know if you would like me to remove all code-folding related translations I have merged into non-root strings. These can certainly be sorted promptly.